### PR TITLE
fix(report-portal): Fixed search for projects page

### PR DESCRIPTION
**Pull request by yashoswalyo**
- Added request to add button on instances page

image
- Updated readme
- Changed breadcrums color to white
  
image

### CODE REVIEW

**Comment by yashoswalyo** [comment]
Email template
image

File: plugins/report-portal/src/components/ReportPortalGlobalPage/ReportPortalGlobalPage.tsx [hunk 1]
```diff
@@ -13,14 +14,38 @@ export type ReportPortalGlobalPageProps = {
...
+          >
+            Don't see your instance here? &nbsp;
+            <Button
+              style={{ textTransform: 'none', padding: '5px 10px' }}
+              color="primary"
+              variant="outlined"
+              target="_blank"
+              href={`https://mail.google.com/mail/?view=cm&fs=1&tf=1&to=${encodeURIComponent(
+                sendAddress,
+              )}&su=Request%20to%20add%20a%20report%20portal%20instance&body=Requesting%20to%20add%20a%20new%20Report%20Portal%20instance%20to%20Red%20Hat%20Experience%20Platform.%0A%0AHostname%3A%0AProject%20Manager%3A%0AInstance%20Admin%20email%3A`}
```
Comment (deshmukhmayur) [inline plugins/report-portal/src/components/ReportPortalGlobalPage/ReportPortalGlobalPage.tsx]: @yashoswalyo I think the email subject and body should be stored in a string variable first.

```ts
const body = encodeURIComponent(`<multiline
email body>
`);
```

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]

File: plugins/report-portal/src/components/ReportPortalGlobalPage/ReportPortalGlobalPage.tsx (hunk 1 above)
Comment (yashoswalyo) [reply plugins/report-portal/src/components/ReportPortalGlobalPage/ReportPortalGlobalPage.tsx]: done

**Review by deshmukhmayur — APPROVED** [review]
LGTM.

**Comment by deshmukhmayur** [comment]
@yashoswalyo the yarn install step seems to be failing. Maybe because you've bumped up the report portal plugin version. 

Can you update the `yarn.lock`?

**Comment by yashoswalyo** [comment]
Tests are failing even after adding `yarn.lock` file

**Comment by riginoommen** [comment]
Try removing yarn.locak, nodemodules and try the tests

**Review by riginoommen — APPROVED** [review]

**Pull request merged**